doc/nrf/includes/sample_dtm_radio_test_fem.txt

```diff
@@ -3,15 +3,18 @@ You can add support for the nRF21540 front-end module (FEM) to the sample.
 To add support for the nRF21540 FEM, build the sample for a board containing nRF21540 FEM like :ref:`nrf21540dk/nrf52840 <zephyr:nrf21540dk_nrf52840>` or create a devicetree overlay file describing how FEM is connected to nRF5 SoC in your device.
 
 .. note::
-   If you use the nRF21540 EK, append ``nrf21540ek`` shield to your build command instructing build system to append the appropriate devicetree overlay file.
+   If you use the nRF21540 EK with SoCs of nRF52 family, append ``nrf21540ek`` shield to your build command instructing build system to append the appropriate devicetree overlay file.
+   If you use the nRF21540 EK with nRF5340 SoC, append the ``nrf21540ek_fwd`` shield to your build command for the application core image and the ``nrf21540ek`` shield to the network core image.
    If you use the nRF21540 DK, build your application for the :ref:`nrf21540dk/nrf52840 <zephyr:nrf21540dk_nrf52840>` board target.
    The devicetree for the nRF21540 DK already contains the required FEM configuration, so you do not need to set an additional build option.
 
 For example, to build the sample from the command line for an nRF5340 DK with an attached nRF21540 EK, invoke the following command within the sample directory:
```

Comment:
This line says 'following command' but there are 3 commands each for different case. Please clarify